app/src/main/res/layout-sw600dp/story_chapter_view.xml

```diff
@@ -54,7 +54,7 @@
         android:importantForAccessibility="no"
         android:padding="8dp"
         android:visibility="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.COMPLETED ? View.GONE : View.VISIBLE}"
-        app:srcCompat="@drawable/ic_lessons_icon_24dp" />
+        app:srcCompat="@drawable/ic_lessons_icon_24dp"/>
     </FrameLayout>
 
     <FrameLayout
```

Comment:
Instroduce stroke for `MaterialCardView ` in dark-mode after applying above changes so that it looks good.

Maybe use something like this:
```
        app:cardBackgroundColor="@color/component_color_lessons_tab_activity_background_color"
        app:cardCornerRadius="4dp"
        app:cardElevation="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @dimen/story_chapter_card_playable_state_elevation : @dimen/story_chapter_card_not_playable_state_elevation}"
        app:rippleColor="@color/component_color_lessons_tab_activity_lesson_card_ripple_color"
        app:strokeColor="@color/component_color_lessons_tab_activity_lesson_card_border_stroke_color"
        app:strokeWidth="1dp">
```

Don't use exact code, because naming of colors is incorrect.

Reply:
Done.

Reply:
Remove `component_color_story_activity_container_background_color` from `ConstraintLayout` and introduce that as `app:cardBackgroundColor` in Card directly.

Reply:
Done.